### ISSUE TITLE
Client ip address

### ISSUE THREAD

**Issue by divyanagrath** [fastify/fastify#1661]
I need to log the client ip address from where the request is coming. How to access that?

Already tried 'request.ip' and 'request.ips'. But it returns server ip.

**Comment by delvedor** [comment]
What do you mean with "server ip"? Are you testing in localhost?

This is how we detect the client ip address:
https://github.com/fastify/fastify/blob/ff5bfddf3dcd828e77fef74de245b414384b63da/lib/route.js#L282-L311

**Comment by divyanagrath** [comment]
No, not in localhost. 'request.ip' returns the server IP address, but not of the client who has made the request. So basically if from my computer I'm making a request to server, then it should return my computer's IP address.

**Comment by Eomm** [comment]
Could you show your code and the logs you get?

**Comment by divyanagrath** [comment]
I have added console.log(request.ip) in 'prehandler' hook.  In logs I get same ip for all the API hits from different clients.
